SparkPost/2web2ui
# SA-1620: Group by table comparison filters

### What Changed

- Group By table swaps for CompareByGroupByTable (if there are comparison filters)
- Column added to comparison filter
- Added new useSparkPostQueries

### How To Test

- ON report builder with compare by enabled, add at least 2 comparison filters
 - Go to group by table and verify table works as expected
 - Verify that clicking filter links still adds filters to table
 - Verify that table is still sortable by column
 - Verify that default sort is by group by column (no longer first column)
- Block one of the comparison filters request from going through (and verify error state)
 - Block all of the comparison filters requests from going through (and verify error state)

### To Do

- [ ] Add testing
 - [ ] Disable option for group by if select column is used (or maybe just disable 2nd row if it's the same?)
